ksql-cli/src/main/java/io/confluent/ksql/cli/commands/AbstractCliCommands.java

```diff
@@ -63,6 +67,11 @@
   )
   String outputFormat = OutputFormat.TABULAR.name();
 
+
+  public AbstractCliCommands() {
```

Comment:
we should have another constructor `AbstractCliCommands(VersionCheckerAgent agent)` and have this one delegate to that. This way we can test the CLI without the version checker etc.